src/index.ts

```diff
@@ -16,35 +16,35 @@ const start = async () => {
   // load in the config
   LoadConfig("config.yaml");
 
-  logger.debug({Config});
+  //logger.debug({Config});
 
   // create client as ClientType
   const client: ClientType = new Client({
-    intents: [Intents.FLAGS.GUILDS],
+    intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES],
   }) as ClientType;
 
   client.commands = new Collection();
 
   // return a string array of file names
   // where the file name ends in `.ts` and it is enabled
   // in featurization
-  const commandFiles: string[] = fs.readdirSync("./src/commands/").filter(
-    (file: string) =>
-      file.endsWith(".ts") && (Config as any)?.features[file.slice(0, -3)] // slice to get rid of extension
-  );
 
-  // dynamically import and load commands
+  // dynamic command loader
+  const commandFiles = fs.readdirSync("./src/commands") //
+  .filter((name) => name.endsWith(".ts"));
+
+    // dynamically import and load commands
   for (const file of commandFiles) {
     const filePath = path.format({
       root: "./commands/",
       name: file,
     });
 
     // actual dynamic import
-    const {command} = await import(filePath.slice(0, -3));
+    const {command} = await import(filePath);
```

Comment:
has loading in the commands been tested? I would expect there to be issues with the extension.